Formula/rrdtool.rb

```diff
@@ -28,6 +28,11 @@ class Rrdtool < Formula
   patch :DATA
 
   def install
+    # fatal error: 'ruby/config.h' file not found
+    if MacOS.sdk && (MacOS.sdk.version != MacOS.version)
```

Comment:
Right, and that conditional would trap that case, too ;)

If someone has Xcode 8, it would be worth checking this one with El Capitan to see if Apple's done anything to improve the situation for the ruby.h issue.